<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1513</width>
    <height>793</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>641</width>
      <height>201</height>
     </rect>
    </property>
    <property name="title">
     <string>Loading</string>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>101</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Image Dir:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>101</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Detection Dir:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>160</y>
       <width>191</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Load</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_imagePath">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>291</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>/home/jhz/workspace/projects/dataset/TUM_test/selected/fr1_desk/images/</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_detection_dir">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>291</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>/home/jhz/workspace/projects/dataset/TUM_test/selected/fr1_desk/associations/</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_num">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>170</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>0 / 0</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_labelConfig">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>60</y>
       <width>291</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>/home/jhz/workspace/projects/dataset_tools/instanceAnnotation/data/coco.names</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>40</y>
       <width>141</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Label config Path:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_instancePath">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>120</y>
       <width>291</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>/home/jhz/workspace/projects/dataset_tools/instanceAnnotation/data/instances.txt</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>100</y>
       <width>141</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Instance Path:</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>861</width>
      <height>501</height>
     </rect>
    </property>
    <property name="title">
     <string>Annotation</string>
    </property>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>300</y>
       <width>101</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Instance Lists</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>101</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Detection List:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>460</y>
       <width>99</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Add</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="tableWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>831</width>
       <height>231</height>
      </rect>
     </property>
     <column>
      <property name="text">
       <string>id</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>x1</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>y1</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>x2</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>y2</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>label</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>prob</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>INSTANCE</string>
      </property>
     </column>
    </widget>
    <widget class="QTableWidget" name="tableWidget_instance">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>320</y>
       <width>281</width>
       <height>131</height>
      </rect>
     </property>
     <column>
      <property name="text">
       <string>id</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>label</string>
      </property>
     </column>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>10</y>
      <width>640</width>
      <height>480</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">border:2px solid red</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>1200</x>
      <y>530</y>
      <width>111</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>500</y>
      <width>301</width>
      <height>71</height>
     </rect>
    </property>
    <property name="title">
     <string>GroupBox</string>
    </property>
    <widget class="QPushButton" name="pushButton_left">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>121</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_right">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>30</y>
       <width>121</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>&gt;</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>1360</x>
      <y>530</y>
      <width>111</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Quit</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1513</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
